frontend/express/libs/members.js

```diff
@@ -543,54 +545,81 @@ membersUtility.extendSession = function(req) {
 
 membersUtility.setup = function(req, callback) {
     membersUtility.db.collection('members').count(function(err, memberCount) {
-        if (!err && memberCount === 0) {
+        if (!err && memberCount === 10) {
```

Comment:
Out of curiosity, why 10? What does this do?

Reply:
Sorry this was for debug purpose. Let me fix it.